### ISSUE TITLE
could not find function "nexprs"

### ISSUE THREAD

**Issue by huidongchen** [davismcc/archive-scater#68]
I have installed package 'scater' and imported it. But the error was reported when I tried to call 'nexprs'.

**Comment by LTLA** [comment]
Are you sure you're using the latest version of _scater_ (1.2.0)? Please post a minimum working example that reproduces the error on your machine, and the session information from `sessionInfo()`.

**Comment by huidongchen** [comment]
Thanks for your quick reply. 

This is my session info:
- R version 3.3.1 (2016-06-21)
- Platform: x86_64-apple-darwin13.4.0 (64-bit)
- Running under: OS X 10.12 (Sierra)

I followed the standard installation.
- source("https://bioconductor.org/biocLite.R")
- biocLite("scater")

The latest version of scater(1.2.0) was supposed to be installed. But when I checked it, it was scater_1.0.4 surprisingly. 

Finally the problem was solved by the _updating_ command.
- source("https://bioconductor.org/biocLite.R")
- biocLite("BiocUpgrade") 

Thanks again for your help. Awesome package!
